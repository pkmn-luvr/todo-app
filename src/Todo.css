.Todo {
    display: flex;
    align-items: center;
    justify-content: space-between;
    padding: 10px;
    margin-bottom: 5px;
    background-color: #e9f1f7;
    border-radius: 5px;
  }
  
  .Todo button {
    margin-left: 10px;
  }
  