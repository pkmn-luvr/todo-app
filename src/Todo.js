import React from 'react';

function Todo({ id, task, removeTodo }) {
  return (
    <div>
      {task}
      <button onClick={() => removeTodo(id)}>X</button>
    </div>
  );
}

export default Todo;
