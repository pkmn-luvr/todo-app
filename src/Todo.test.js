import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

describe('Todo Component Tests', () => {
  // Smoke
  test('renders without crashing', () => {
    render(<Todo task="Test Task" />);
  });

  // Snapshot
  test('matches snapshot', () => {
    const { asFragment } = render(<Todo task="Test Task" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
