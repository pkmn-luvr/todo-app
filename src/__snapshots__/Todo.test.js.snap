// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Component Tests matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Test Task
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
